.github/workflows/ci.yml

```diff
@@ -78,7 +82,11 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - uses: ./.github/commands/dependencies/install_deps
+        uses: ./.github/commands/dependencies/install_deps
+      - name: Download Build 
+        uses: actions/download-artifact@v4
+        with:
+          name: build-artifact
```

Comment:
I believe this will download to the right place

Reply:
<img width="483" height="846" alt="Screenshot 2025-08-01 at 12 38 35 PM" src="https://github.com/user-attachments/assets/069c10af-82bc-41b1-9594-5ff7072ee0a7" />

I have confirmed that it downloads to the right place haha